website/components/pluginData.tsx

```diff
@@ -67,6 +67,13 @@ export const SOURCE_PLUGINS: Plugin[] = [
     id: "gandi",
     kind: "official",
   },
+  {
+    name: "Google Analytics",
+    logo: "/images/logos/plugins/ga.svg",
```

Comment:
Can we delete `googleanalytics.svg` and `googlenanalytics-dark.svg` if not used?

Reply:
done